### ISSUE TITLE
Perhaps consider replace `geometric_median_dataset` with `nn.BatchNorm1d(x, affine=False)`

### ISSUE THREAD

**Issue by wassname** [ai-safety-foundation/sparse_autoencoder#179]
Perhaps consider replacing `geometric_median_dataset` with `nn.BatchNorm1d(x, affine=False)`
 layer
It doe a similar thing with less parameters and complexity.
